.github/workflows/model_card_consistency_reminder.yml

```diff
@@ -0,0 +1,39 @@
+name: Model and Dataset Card consistency reminder
+
+on:
+  pull_request:
+    paths:
+      - modelcard.md
+      - datasetcard.md
+      - docs/hub/datasets-cards.md
+      - docs/hub/model-cards.md
+      - docs/hub/model-card-annotated.md
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find doc comment
+        uses: peter-evans/find-comment@v2
+        id: find_comment
+        with:
+          issue-number: ${{ steps.github-context.outputs.pr_number }}
+          body-includes: docs for this PR
+
+      - name: Add doc comment if not present
+        uses: thollander/actions-comment-pull-request@v2
+        if: steps.find_comment.outputs.comment-id == ''
+
+        with:
+          message: |
+            It looks like you've updated documentation related to model or dataset cards in this PR.
+            Please make sure that the following files stay consistent between them:
```

Comment:
I'm curious. They should stay consistent in terms of what? content, yaml modelcard, or something else?

Reply:
Mostly in term of content. One is the description of what should be in a model card, one is an example of model card and one is an example of model card where each field is annotated. So definitely some redundancy that should stay consistent. (and about why having those, it's just that they have different purposes in term of documentation).

Reply:
In this case, could you update the wording a bit to be more precise rather than:
> Please make sure that the following files stay consistent: